build-concourse/build-corgi.ts

```diff
@@ -146,10 +155,34 @@ function makePipeline(env: KeyValue) {
         })
     }
 
+    const buildGitPptxJob = (resource: RESOURCES, tasks: any[]) => {
+        const report = reportToCorgi(resource)
+        const lookupBookDef = buildLookUpBook(resource)
+        // PDF_OR_WEB argument does not seem to actually do anything
+        const lookupBookTask = taskMaker(env, PDF_OR_WEB.PDF, lookupBookDef)
```

Comment:
Is it not supposed to be `PDF_OR_WEB.WEB` ?

Reply:
That argument does not do anything. I am not sure why it's still around except that no one made time to remove it. It's part of openstax/ce#1953. I will do it as part of this PR since it is causing confusion.